webpack.config.js

```diff
@@ -0,0 +1,33 @@
+var path = require('path');
```

Comment:
we'll need to update this to include a production version as well, but can consider that out of scope.
if you want to reach for it, you could look at my use of env.dev here: 
https://github.com/wcjordan/devtool-boilerplate/blob/master/shells/chrome/webpack.config.js

it's a bit cleaner than FDT or LD